### PR TITLE
add test_provider and test_requirer files to template

### PR DESCRIPTION
**Pull request by PietroPasotti**
adds to `__template__` folder the test_provider and test_requirer python files (empty)

### CODE REVIEW

**Review by arturo-seijas — APPROVED** [review]

**Comment by jnsgruk** [comment]
Might be worth linking a a small doc page or an example of some interface tests in the comment? :)

**Comment by PietroPasotti** [comment]
> Might be worth linking a a small doc page or an example of some interface tests in the comment? :)
Added a readme pointing to the docs as well as providing a minimal quickstart

**Comment by PietroPasotti** [comment]
@jnsgruk or @simskij ?

**Review by gruyaume — APPROVED** [review]

File: interfaces/__template__/v0/interface_tests/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+from scenario import Relation, State
+from interface_tester import Tester
+
+def test_data_published_on_changed_remote_valid():
+    # GIVEN a relation over <interface> containing all the right data 
+    #  in the right format for the <requirer side>:
+    relation = Relation(
+        endpoint='ingress',
+        interface='ingress', remote_app_name='remote',
+        remote_app_data={'foo': '"bar"', 'baz': '42', 'qux': '"www.lol.com:4242"', },
```
Comment (gruyaume) [inline interfaces/__template__/v0/interface_tests/README.md:21]: Are the added quotes intentional in `bar` and `www.lol...`?

**Pull request merged**